tests/gtests/sw_testhelpers.h

```diff
@@ -8,6 +8,8 @@
 static const double
   tol0 = 1e-0, tol1 = 1e-1, tol2 = 1e-2, tol3 = 1e-3, tol6 = 1e-6, tol9 = 1e-9;
 
+extern const char * masterfile_test = "files.in"; // relative to 'dir_test'

```

Comment:
`*masterfile_test` is currently neither defined nor declared -- so the extern doesn't work here. Maybe remove the "extern" keyword